extensions/mssql/src/dacfx/dacFxService.ts

```diff
@@ -119,4 +119,15 @@ export class DacFxService implements mssql.IDacFxService {
 			}
 		);
 	}
+
+	public validateStreamingJob(packageFilePath: string, createStreamingJobTsql: string): Thenable<mssql.ValidateStreamingJobResult> {
```

Comment:
should the return type here also use Promise instead of Thenable?

Reply:
They should... but in a different PR I think, 'cause this is a more critical path that I don't want to change in askmode, just in case there's some slight difference between Thenable and Promise that changes behavior.